item_list.txt

```diff
@@ -18,39 +18,3 @@ Sound, Speaker, JBL Vrx
 Sound, Speaker, JBL Vrx
 Sound, Speaker, JBL Vrx
 Sound, Speaker, JBL Vrx
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Stage, Podest, 2x1
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Beam, Gladiator
-Light, Kablarka, DMX
-Power, Kablarka, pr�dowa du�a
```

Comment:
In this and following commits you append changes to this "database" file, I assume its because of you testing changes? If so, it would be good to not to add these changes (manually remove it before committing or maybe even add this file to .gitignore) to not to bloat commit diff message, what do you think?

Reply:
You are right, I will add this file to .gitignore :)